brainspell/article_helpers.py

```diff
@@ -96,56 +95,65 @@ def toggle_vote(pmid, topic, username, direction):
                      article_id=pmid,
                      vote=direction,
                      type=True).execute()
-        if direction: # Increase number of agreements
-            Tags.update(agree = Tags.agree + 1)\
+        if direction:  # Increase number of agreements
+            Tags.update(agree=Tags.agree + 1)\
                 .where((Tags.tag_name == topic) &
                        (Tags.article_id == pmid) &
-                       (Tags.experiment_id == None)
+                       (Tags.experiment_id is None)
                        ).execute()
         else:
-            Tags.update(disagree = Tags.disagree + 1).\
+            Tags.update(disagree=Tags.disagree + 1).\
                 where((Tags.tag_name == topic)
                       & (Tags.article_id == pmid)
-                      & (Tags.experiment_id == None)
+                      & (Tags.experiment_id is None)
                       ).execute()
     elif target.count > 0:
         target = next(target)
-        if target.vote == direction: # Remove the vote (constitutes a double click)
-            Votes.delete().where((Votes.username==username) &
+        # Remove the vote (constitutes a double click)
+        if target.vote == direction:
+            Votes.delete().where((Votes.username == username) &
                                  (Votes.name == topic) &
                                  (Votes.article_id == pmid)).execute()
-            # Remove the vote from the agree/disagree fields of Articles Mesh tags
-            if direction: # If vote is an agreement, decrement the agreement field
-                Tags.update(agree = Tags.agree - 1).where((Tags.tag_name == topic)
-                                                          & (Tags.article_id == pmid)
-                                                          & (Tags.experiment_id == None)
-                                                          ).execute()
+            # Remove the vote from the agree/disagree fields of Articles Mesh
+            # tags
+            if direction:  # If vote is an agreement, decrement the agreement field
+                Tags.update(
+                    agree=Tags.agree -
```

Comment:
just curious: why split the line here ? are you using an automatic tool ?

Reply:
Yup, the autopep8 python style tool that we're using formats everything automatically. (Sometime's weirdly). I'll probably try and make everything look reasonable once all this is done.

Reply:
sounds good !